content/article/cardano-cube-find-every-project-thats-building-on-the-cardano-blockchain.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Cardano Cube - Find every project that's building on the Cardano Blockchain
```

Comment:
Can we please slightly tweak the title to:
Cardano Cube - find every project that's building on the Cardano blockchain